src/Backend/Modules/Location/Actions/Edit.php

```diff
@@ -95,6 +95,7 @@ private function loadData(): void
         }
 
         $this->settings['full_url'] = (isset($this->settings['full_url'])) ? ($this->settings['full_url']) : false;
+        $this->settings['add_map_link_to_address'] = (isset($this->settings['add_map_link_to_address'])) ? ($this->settings['add_map_link_to_address']) : false;
```

Comment:
isn't this just `$this->settings['add_map_link_to_address'] = $this->settings['add_map_link_to_address'] ??  false;`